openedx/core/djangoapps/schedules/admin.py

```diff
@@ -122,8 +122,8 @@ def choices(self, changelist):
 
 @admin.register(models.Schedule)
 class ScheduleAdmin(admin.ModelAdmin):
-    list_display = ('username', 'course_id', 'active', 'start', 'upgrade_deadline', 'experience_display')
-    list_display_links = ('start', 'upgrade_deadline', 'experience_display')
+    list_display = ('username', 'course_id', 'active', 'start', 'start_date', 'upgrade_deadline', 'experience_display')
+    list_display_links = ('start', 'start_date', 'upgrade_deadline', 'experience_display')
```

Comment:
This change should also be removed since we do not want both start and start_date visible.